bigbluebutton-html5/private/static/guest-wait/guest-wait.html

```diff
@@ -364,7 +402,7 @@ <h1 id="heading">BigBlueButton - Guest Lobby</h1>
     </div>
     <p aria-live="polite">Please wait for a moderator to approve you joining the meeting.</p>
     <div id="positionInWaitingQueue">
-      <p aria-live="polite">Calculating position in waiting queue</p>
+      <p aria-live="polite">Retrieving user's status.</p>
```

Comment:
Position in waiting queue was quite descriptive, to be honest...
'Status' could be 'away' or 'connecting', but not the same as position.

As an aside, do these strings need to be included in (and picked up from) en.json?